Plugin/Admin/Controller/PaymentsController.php

```diff
@@ -220,7 +220,7 @@ public function add()
             $this->log($message);
             die(json_encode(array('status'=>0,'msg'=>$message)));
         }
-
+        
```

Comment:
hooks in eclipse / windows do not work despite setting path var as mentioned here: http://zaunerc.blogspot.co.at/2015/11/git-hooks-currently-supported-by-jgit.html

i try always to use command line, but in this case i used the much more convenient egit gui. i try not to forget to use command line, but i am very used to use the gui tool, so this may last a while :-)

eclipse egit on windows should support the pre-commit-hook. no idea why it does not.